etc/licscripts/find_invlic.pl

```diff
@@ -71,9 +83,12 @@ ()
       return;
     }
   }
-
-  printf("** File %s did NOT match any license\n", $File::Find::dir . "/" . $entry);
-  $ok = 0;
+  if ( ! $use_known_files or File::Spec->rel2abs($entry) ~~ @known_files) { 
+    printf("** File %s did NOT match any license\n", $File::Find::dir . "/" . $entry);
+    $ok = 0;
+  } else {
+    printf("** File %s did NOT match any license, but is not known to git!\n", $File::Find::dir . "/" . $entry);
```

Comment:
The current output is somewhat irritating, as it first complains about a missing license, and then claims that all files are properly licensed. You should inform the user that the file is ignored:
```suggestion
    printf("** File %s did NOT match any license, but is not known to git, ignoring!\n", $File::Find::dir . "/" . $entry);
```